src/Stratis.Bitcoin.Features.Wallet/Controllers/WalletController.cs

```diff
@@ -705,28 +703,29 @@ public IActionResult BuildTransaction([FromBody] BuildTransactionRequest request
             try
             {
                 Script destination = BitcoinAddress.Create(request.DestinationAddress, this.network).ScriptPubKey;
-                var context = new TransactionBuildContext(
-                    this.network,
+
+                var txBuildOptions = new TransactionBuildOptions(
```

Comment:
If I made the rule it would roughly be:

Class/interface names must use Transaction. 
Fields / variables / properties may use tx or Transaction.

Some lines are too long with spelling out Transaction every time, and I would try to discourage use of txn.

Happy to amend here if you wish.

Reply:
Nah, don't amend, let's keep tx since it is used in the codebase a lot. Just food for thought.